src/NetAppFiles/NetAppFiles/help/az.NetAppFiles.md

```diff
@@ -0,0 +1,55 @@
+---
+Module Name: az.NetAppFiles
```

Comment:
@leonardbf please make sure this file is renamed to `Az.NetAppFiles.md` and that all instances of `az.NetAppFiles` are changed to `Az.NetAppFiles` in this file

Reply:
ok. Done.